src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**Storage**
+
+* Add a new command group `az storage share-rm` to use the Microsoft.Storage resource provider for Azure file share management operations.
```

Comment:
What about "Add a new command group `az storage share-rm` to support manage File share (create/delete/update/show/list) in mgmt plane"?

Reply:
CLI users should not care if it's mgmt plane or data plane

---
In reply to: [364247847](https://github.com/Azure/azure-cli/pull/11649#discussion_r364247847) [](ancestors = 364247847)